docs/operations.md

```diff
@@ -18,10 +18,54 @@ then deleting the old chunks. The scavenged events are also removed from the ind
 
 Running a scavenge causes the active chunk to be completed so that it can be scavenged.
 
-::: warning 
+::: warning
 Scavenging is destructive. Once a scavenge has run, you cannot recover any deleted events except from a backup.
 :::
 
+### Mechanism of the scavenging
```

Comment:
I think it would probably make sense to have the section explaining the mechanism after the general information about how to run it